openeo_udp/worldcereal_crop_extent.json

```diff
@@ -1291,27 +1320,13 @@
       }
     },
     {
-      "name": "orbit_state",
-      "description": "The orbit state of the Sentinel-1 data",
+      "name": "projection",
+      "description": "The EPSG code of the output projection",
       "schema": {
-        "type": "string",
-        "enum": [
-          "ASCENDING",
-          "DESCENDING"
-        ]
+        "type": "string"
       },
-      "default": "DESCENDING",
+      "default": null,
       "optional": true
     }
-  ],
-  "minimal_job_options": {
```

Comment:
does it work without job options?

Reply:
Currently, the only way it works, is to specify job options after calling the UDP